BlueprintUI/Sources/Layout/Transformed.swift

```diff
@@ -0,0 +1,134 @@
+import UIKit
+
+/// Changes the transform of the wrapped element.
+public struct Transformed: Element {
+
+    /// The content element whose transform is being affected.
+    public var wrappedElement: Element
+
+    /// The transform of the wrapped element.
+    public var transform: CATransform3D
+
+    /// Initializes a `Transformed` with the given content element and 3D transform.
+    ///
+    /// - parameters:
+    ///   - transform: The 3D transform to be applied to the wrapped element.
+    ///   - wrapping: The content element to be made transparent.
+    public init(
+        transform: CATransform3D,
+        wrapping wrappedElement: Element
+    ) {
+        self.transform = transform
+        self.wrappedElement = wrappedElement
+    }
+
+    /// Initializes a `Transformed` with the given content element and DD transform.
+    ///
+    /// - parameters:
+    ///   - transform: The 2D transform to be applied to the wrapped element.
+    ///   - wrapping: The content element to be made transparent.
+    public init(
+        transform: CGAffineTransform,
+        wrapping wrappedElement: Element
+    ) {
+        self.transform = CATransform3DMakeAffineTransform(transform)
+        self.wrappedElement = wrappedElement
+    }
+
+
+    public var content: ElementContent {
+        return ElementContent(child: wrappedElement, layout: Layout(transform: transform))
+    }
+
+    public func backingViewDescription(bounds: CGRect, subtreeExtent: CGRect?) -> ViewDescription? {
+        return nil
+    }
+
+    private struct Layout: SingleChildLayout {
+        var transform: CATransform3D
+
+        func measure(in constraint: SizeConstraint, child: Measurable) -> CGSize {
+            return child.measure(in: constraint)
```

Comment:
Does this measure against the transformed element? I don't think it will; I wonder if we want to apply the transform to the measured size/bounds?

Reply:
But then again, I guess that would affect the frame of reference of the layout, which isn't desirable.